.github/workflows/docs.yml

```diff
@@ -44,6 +44,15 @@ jobs:
             });
             core.setOutput('head_ref', pr.head.ref);
 
+      - name: Comment That docs building is starting
```

Comment:
Can we add back the check for [docs] in commit string please? It helps for folks making PR from forks.